src/ServiceControl/Bootstrapper.cs

```diff
@@ -71,10 +71,19 @@ public Bootstrapper(ServiceBase host = null, HostArguments hostArguments = null,
 
             configuration.UseSerialization<JsonSerializer>();
 
-            configuration.Transactions()
-                .DisableDistributedTransactions()
-                .DoNotWrapHandlersExecutionInATransactionScope();
-            
+            if (Settings.EnableDtc)
+            {
+                configuration.Transactions()
+                    .EnableDistributedTransactions()
+                    .WrapHandlersExecutionInATransactionScope();
+            }
+            else
+            {
+                configuration.Transactions()
+                    .DisableDistributedTransactions()
+                    .DoNotWrapHandlersExecutionInATransactionScope();
+            }
+
```

Comment:
I went for being explicit rather than relying on defaults. Lines 75 and 76 were the prior behavior which is the default.